### DISCUSSION TITLE
Tab completion, `components` go to crazy places

### DISCUSSION THREAD

**Discussion by JoeCianflone** [vuejs/vetur#2897] [Q&A]
I'm assuming I've done something wrong when setting up Vuter, but what...I have no idea. Importing seems to work, it correctly finds all my components, but, well, it doesn't add the component to the `component: {}` area it kinda just dumps it randomly on my page.  I can't fully reproduce this because sometimes it works. Is there a way to say, "this is where the components property is located"
<img width="416" alt="Screen Shot 2021-04-27 at 10 40 09 AM" src="https://user-images.githubusercontent.com/989648/116261564-c4b96b00-a745-11eb-938c-a05655f97516.png">
You can see in this screenshot what I'm talking about I imported that `FormField` component and that's the right path, but it should be `components: {FormField,},` and it's not. 

Anyone see this before?

**Comment by JoeCianflone** [comment]
Point of note: I'm talking about tab completion. If I'm in the `<template>` area and start typing `<FormButton>` and use the tab completion...the import is correctly placed at the top of the page, but the `components` name gets dropped randomly on my page. I think this is a setting, but I have no idea where or what to change exactly

**Comment by yoyo930021** [comment]
It's a bug.
Follow https://github.com/vuejs/vetur/issues/2501.
